### ISSUE TITLE
Initialize a version from its string representation should be Optional

### ISSUE THREAD

**Issue by twodayslate** [mrackwitz/Version#44]
`Version("")` will crash the device. Shouldn't it be `init?` instead of `init!`

```
Error: Failed to parse version number '': InvalidComponents
Fatal error: Unexpectedly found nil while unwrapping an Optional value
2019-09-11 21:05:13.709726-0400 ec3730[79029:21121642] Fatal error: Unexpectedly found nil while unwrapping an Optional value
```

**Comment by twodayslate** [comment]
This has been completed in #45 - a new release still has to be cut tho

**Comment by regexident** [comment]
Resolved with 0.7.1
